packages/xsnap/src/xsnap.c

```diff
@@ -822,24 +844,20 @@ void fxPrintUsage()
 // 	xsResult = (the->promiseJobs) ? xsFalse : xsTrue;
 // }
 
-// void fx_print(xsMachine* the)
-// {
-// 	xsIntegerValue c = xsToInteger(xsArgc), i;
-// 	if (gxMeteringPrint)
-// 		fprintf(stdout, "[%u] ", the->meterIndex);
-// 	for (i = 0; i < c; i++) {
-// 		if (i)
-// 			fprintf(stdout, " ");
-// 		fprintf(stdout, "%s", xsToString(xsArg(i)));
-// 	}
-// 	fprintf(stdout, "\n");
-// }
+void fx_print(xsMachine* the)
+{
+	xsIntegerValue c = xsToInteger(xsArgc), i;
+	if (gxMeteringPrint)
+		fprintf(stdout, "[%u] ", the->meterIndex);
+	for (i = 0; i < c; i++) {
+		if (i)
+			fprintf(stdout, " ");
+		fprintf(stdout, "%s", xsToString(xsArg(i)));
+	}
+	fprintf(stdout, "\n");
+		fflush(stdout);
```

Comment:
nit: indent alignment

Reply:
I blame it on insufficient tool support. You'll excuse me if I don't spend yet another CI cycle on this.